core/src/test/java/org/apache/iceberg/TestManifestWriter.java

```diff
@@ -93,8 +93,7 @@ public void testManifestPartitionStats() throws IOException {
   @TestTemplate
   public void testWriteManifestWithSequenceNumber() throws IOException {
     assumeThat(formatVersion).isGreaterThan(1);
-    File manifestFile = File.createTempFile("manifest", ".avro", temp.toFile());
```

Comment:
I see a few more places around core that use the same pattern. Could you please update those as well?

<img width="962" height="528" alt="image" src="https://github.com/user-attachments/assets/9d63742e-ca2e-41bb-bce7-eba58ef8abb0" />

Reply:
can you also please update `DeleteReadTests` in `data`?

Reply:
@nastra Done. There were many other tests as well.